.irbrc

```diff
@@ -1 +1,2 @@
 IRB.conf[:USE_AUTOCOMPLETE] = false
+IRB.conf[:SAVE_HISTORY] = nil
```

Comment:
I like this during development, so it'd be nice to keep it if possible. Is the history saved somewhere in the repository's directory?

Reply:
The history for me doesn't end up in the project directory, could we add the file to `.gitignore` as an alternative?

Reply:
If I had to guess, it's normally saved at `~/.irb_history`, but the addition of a `.irbrc` in #7770 caused it to start being saved in the project root instead. I don't feel strongly about allowing history and just adding the file to `.gitignore` if you'd prefer to have it. Another possible option could be to set it back to `~/.irb_history`.

Reply:
> The history for me doesn't end up in the project directory, could we add the file to `.gitignore` as an alternative?

🤔 Interesting. Are you running `irb` in a branch that's been updated since #7770 ?

Reply:
My worry with specifying an explicit directory is it could cause issues in deployed environments with permissions, etc.

Reply:
> > The history for me doesn't end up in the project directory, could we add the file to `.gitignore` as an alternative?
> 
> 🤔 Interesting. Are you running `irb` in a branch that's been updated since #7770 ?

Yep.

Reply:
Ok, I'll just leave the default configuration and add the file to `.gitignore`

Reply:
> Ok, I'll just leave the default configuration and add the file to `.gitignore`

Updated in 5b052de77.

Reply:
I think it may depend on the Ruby installation method. Mine ends up in `~/.irb_history`, not sure why.